src/javascript/JContent/ContentRoute/ToolBar/ToolBar.jsx

```diff
@@ -31,22 +31,24 @@ export const ToolBar = () => {
                 <BrowseControlBar showActions={selection.length === 0}/>}
             {selection.length > 0 &&
             <React.Fragment>
-                <Typography variant="caption" data-cm-role="selection-infos" data-cm-selection-size={selection.length}>
-                    {t('jcontent:label.contentManager.selection.itemsSelected', {count: selection.length})}
-                </Typography>
-                <div className={styles.spacer}/>
-                <Button icon={<Cancel/>} variant="ghost" size="small" onClick={() => dispatch(cmClearSelection())}/>
+                <div className={`flexRow ${styles.selection}`}>
+                    <Typography variant="caption" data-cm-role="selection-infos" data-cm-selection-size={selection.length}>
+                        {t('jcontent:label.contentManager.selection.itemsSelected', {count: selection.length})}
+                    </Typography>
+                    <div className={styles.spacer}/>
+                    <Button icon={<Cancel/>} variant="ghost" size="small" onClick={() => dispatch(cmClearSelection())}/>
+                </div>
                 <div className="flexRow">
                     <Separator variant="vertical" invisible="onlyChild"/>
                     <DisplayActions
                         target="selectedContentActions"
                         context={{paths: selection}}
                         filter={action => action.key !== 'deletePermanently' && action.key.indexOf('publish') === -1}
-                        render={getButtonRenderer({size: 'small', variant: 'ghost'})}
+                        render={getButtonRenderer({size: 'default', variant: 'ghost'})}
                     />
                 </div>
                 <Separator variant="vertical" invisible="onlyChild"/>
```

Comment:
You don't have the "edit" button when selection is on one item ? Is it still required ? (still there in the mockups)

Reply:
we had that in cmm before? As far as I know 'edit' is not part of multiple selection even if it's 1 item, maybe I'm wrong

Reply:
No, never had that in CMM. But it's part of jcontent mockup ? Anyway, that will require changes in content editor